core/src/main/java/org/apache/calcite/prepare/CalciteCatalogReader.java

```diff
@@ -303,7 +303,7 @@ public static SqlOperatorTable operatorTable(String... classNames) {
   }
 
   /** Converts a function to a {@link org.apache.calcite.sql.SqlOperator}. */
-  private static SqlOperator toOp(SqlIdentifier name,
+  private SqlOperator toOp(SqlIdentifier name,
```

Comment:
here as well

Reply:
Since we need to call the non-static `config` field inside the toOp method to determine `config.conformance().allowNiladicParentheses()`, the toOp method cannot be a static method.